### ISSUE TITLE
similar idea

### ISSUE THREAD

**Issue by stax76** [oleg-shilo/shell-x#1]
Hello,

I just want to let you know that I believe your extension is similar to mine:

https://github.com/stax76/OpenWithPlusPlus

There is another one (unmaintained) called Open++ which is almost identical to mine but I had this idea independently.

Did you know either when you developed it? Both are over a decade old.

Shell extensions are best done in C++ (Raymond Chen).

My media player mpv.net uses your cs-script.

https://github.com/stax76/mpv.net

Greetings

**Comment by oleg-shilo** [comment]
Hi there, 

Yes the extension is very old. I have implemented it first probably about 13 years ago when I did CS-Script. Yes, that long :) I think it was done originally in C++. All versions of CS-Script were shipped with it.  

I haven't touched it all that time. Though I recently tried to update it and realized that I have lost the original source code. :( 
But since it was a very simple extension I have just re-implemented it again but in C#.

Yep, the idea is quite similar. And if it came to a few people independently it means that it is a good one :)

Though in my case I was really aiming for zero-configuration approach that I borrow from Windows "send-to" implementation. This is probably the only difference between yours and mine. But the dynamic creation of the items is the same.
